src/pudl/extract/eia860.py

```diff
@@ -71,6 +71,8 @@ def get_dtypes(page, **partition):
     "raw_eia860__cooling_equipment",
     "raw_eia860__emission_control_strategies",
     "raw_eia860__emissions_control_equipment",
+    "raw_eia860__energy_storage_existing",
+    "raw_eia860__energy_storage_retired_canceled",
```

Comment:
Do you think that the word `generator` should be added to this table name?? Seems kind of strange because storage isn't technically a generator, even though it has a generator id...

Reply:
I think these are generators both in EIA's lexicon and in a general sense. These are special cases of generators.

Reply:
Ok I can do this and then specify in the table/field description the generic nature of the term generator